src/Parser.js

```diff
@@ -137,31 +142,25 @@ Parser.prototype.parseExpression = function(breakOnInfix, breakOnToken) {
     // we reached the end, a }, or a \right)
     while (true) {
         var lex = this.nextToken;
-        var pos = this.pos;
         if (endOfExpression.indexOf(lex.text) !== -1) {
             break;
         }
         if (breakOnToken && lex.text === breakOnToken) {
```

Comment:
We should probably rename `breakOnToken` to `breakOnChar` so that the word "token" doesn't become too overloaded.